codegenerator/cli/templates/static/codegen/src/eventFetching/ChainFetcher.res

```diff
@@ -306,18 +317,30 @@ let updateFetchState = (
   ~fetchedEvents,
   ~currentBlockHeight,
 ) => {
+  let newItems = switch self.processingFilters {
+  | None => fetchedEvents
+  | Some(processingFilters) => fetchedEvents->applyProcessingFilters(~processingFilters)
```

Comment:
I guess there's no real reason to keep this as an option type, we can just match on empty or not.

Reply:
Yeah. I'll change if I touch the code again